src/NHibernate.Test/NHSpecificTest/GH1963/Fixture.cs

```diff
@@ -50,7 +45,7 @@ where e.Flag
 							 select e;
 
 				Assert.That(
-					result.ToList,
+					() => result.ToList(),
```

Comment:
This was throwing exception that delegate is no valid. Changed this from a method group to a delegate and it triggered an async test generation.